.github/workflows/push-event.yml

```diff
@@ -28,6 +28,23 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
+      - name: Setup Ruby
+        if: ${{ github.repository == 'fossasia/pslab-android' }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
+          bundler-cache: true
+
+      - name: Prepare Bundler
+        if: ${{ github.repository == 'fossasia/pslab-android' }}
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+
+      - name: Pre Checks
```

Comment:
This might not work for pslab. Maybe remove it for now and we can add it later. I can explain what this does